client/asset/zec/zec.go

```diff
@@ -173,6 +173,7 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, net dex.Network) (ass
 		Segwit:                   false,
 		OmitAddressType:          true,
 		LegacySignTxRPC:          true,
+		LegacySentRawTx:          true,
```

Comment:
:+1: confirmed that zcashd is legacy w.r.t. this:

`sendtoaddress "zcashaddress" amount ( "comment" "comment-to" subtractfeefromamount )`